File: ts/packages/over-redis/src/keys.ts
export function commandStream(prefix: string, service: string): string {
  return `${prefix}:cmds:${service}`;
}

export function replyStream(prefix: string, instanceId: string): string {
  return `${prefix}:replies:${instanceId}`;
}

export function consumerGroupName(service: string): string {
  return service;
}

export function consumerName(service: string, instanceId: string): string {
  return `${service}:${instanceId}`;
}

File: py/packages/over-redis/src/clamator_over_redis/keys.py
def command_stream(prefix: str, service: str) -> str:
    return f"{prefix}:cmds:{service}"


def reply_stream(prefix: str, instance_id: str) -> str:
    return f"{prefix}:replies:{instance_id}"


def consumer_group_name(service: str) -> str:
    return service


def consumer_name(service: str, instance_id: str) -> str:
    return f"{service}:{instance_id}"
